Octokit/Models/Response/Issue.cs

```diff
@@ -64,7 +64,7 @@ public Issue(Uri url, Uri htmlUrl, Uri commentsUrl, Uri eventsUrl, int number, I
         public int Number { get; protected set; }
 
         /// <summary>
-        /// Whether the issue is open or closed.
+        /// Whether the issue is "open", "closed" or "all".
```

Comment:
If we're going to `[Obsolete]` `All` here, or just remove it, we should update the docs accordingly...